04-onion-routing.md

```diff
@@ -289,10 +289,11 @@ using an alternate channel.
 
 ## Payload for the Last Node
 
-When building the route, the origin node MUST use a payload for
-the final node with the following values:
+When building the route, the origin node MUST fill the `per_hop` field for the
+final node with the following values:
 * `outgoing_cltv_value`: set to the final expiry specified by the recipient
 * `amt_to_forward`: set to the final amount specified by the recipient
+* `padding`: set to 20 zero bytes
```

Comment:
`padding` is only 12 bytes, though?

The short_channel_id field is left unspecified, but c-lightning sets it to 0.

Reply:
Fixed it